### PR TITLE
docs: logo terms

### LINKED ISSUE #4: Add terms of use for web3community assets
Something like this: https://github.com/logos

### PR DESCRIPTION
**Pull request by krishguptadev**
fixes #4

### CODE REVIEW

**Review by vinzvinci — APPROVED** [review]
LGTM 👍

**Review by shuklaritvik06 — APPROVED** [review]
LGTM 🎉

File: README.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
+# Branding
+
+## Terms
+
+| Do's                                                         | Don'ts                                                       |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| :heavy_check_mark: Use the web3community logo to link to web3community | :x: Do not Use the Web3Community logo for your application’s icon |
+| :heavy_check_mark: Use the web3community logo in a blog post or news article about web3community | :x: Do not Use/Sell any Web3Community artwork without permission |
```
Comment (Panquesito7) [inline README.md:8]:
```suggestion
| :heavy_check_mark: Use the Web3Community logo to link to Web3Community | :x: Do not Use the Web3Community logo for your application’s icon |
| :heavy_check_mark: Use the Web3Community logo in a blog post or news article about Web3Community | :x: Do not Use/Sell any Web3Community artwork without permission |
```

**Review by Panquesito7 — COMMENTED** [review]
I'll make another PR to improve the `README.md` file. 🙂

File: README.md (hunk 1 above)
Comment (krishguptadev) [reply README.md:8]: You can make a new pr based on this since, I can't merge these changes now...

**Pull request merged**